CMakeLists.txt

```diff
@@ -338,6 +326,9 @@ endif()
 find_package(Boost 1.58.0 REQUIRED COMPONENTS serialization iostreams)
 include_directories(${Boost_INCLUDE_DIRS})
 
+find_package(LLVM REQUIRED CONFIG)
+target_include_directories(LLVMSupport INTERFACE ${LLVM_INCLUDE_DIRS})
```

Comment:
Weird that they don't do this for us already, but that's not really our problem to fix.